inventory/prod.yml

```
@@ -93,3 +92,6 @@ all:
         hetzner1.openup.org.za:
       vars:
         app_domain: search.opengazettes.org.za
+    open-justice:
+      hosts:
+        openjustice.openup.org.za:
```

this should be justice1-hetzner.openup.org.za - we use the machine name here, so it's a bit clearer which machine we're talking about
```suggestion
        justice1-hetzner.openup.org.za:
```